stdlib/3/dbm/dumb.pyi

```diff
@@ -0,0 +1,31 @@
+
+import sys
+from typing import Union, MutableMapping, Iterator, Optional, Type
+from types import TracebackType
+
+if sys.version_info >= (3, 8):
+    from typing import Final
+else:
+    from typing_extensions import Final
+
+_KeyType = Union[str, bytes]
+_ValueType = Union[str, bytes]
+
+error: Final[Type[OSError]]
+
+class _Database(MutableMapping[_KeyType, _ValueType]):
```

Comment:
`_ValueType` is not quite right for the value because (as far as I can tell from the code) all values returned from the database are bytes, not str.